packages/protocol/contracts/common/interfaces/IBlocker.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity >=0.5.13 <0.9.0;
+
+interface IBlocker {
```

Comment:
should there be a `block()` function here?

Reply:
no because that's internat implementation of the blocker contract